tests/acceptance/features/apiSpaces/spaceManagement.feature

```diff
@@ -89,118 +89,6 @@ Feature: Space management
     And the json responded should not contain a space with name "Project"
     And the json responded should not contain a space with name "Alice Hansen"
 
-
-  Scenario: The space admin user changes the quota of the project space
```

Comment:
deleted?

Reply:
These scenario got duplicated as it exits on setQuota.feature

Reply:
Please, add comment here with the links so that other can understand

Reply:
The deleted scenarios are covered on `setQuota.feature` file
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L94
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L134
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L50
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L14